packages/x-components/src/x-modules/facets-next/store/__tests__/getters.spec.ts

```diff
@@ -192,15 +202,60 @@ describe('testing facets module getters', () => {
         createNextEditableNumberRangeFilter('age', { min: null, max: 5 })
       ]);
 
-      expect(store.getters.filtersByFacet).toEqual({
+      expect(store.getters.selectedFiltersByFacet).toEqual({
         age: [store.state.filters['age:*-5']]
       });
     });
 
+    it('returns raw filters', () => {
+      const store = createFacetsStore([createRawFilter('size:xl')]);
+
+      expect(store.getters.selectedFiltersByFacet).toEqual({
+        ['__unknown-facet__']: [store.state.filters['size:xl']]
+      });
+    });
+  });
+
+  describe('facets getter', () => {
+    it('returns an empty object if there are no facets', () => {
+      const store = createFacetsStore([]);
+      expect(store.getters.facets).toEqual({});
+    });
+
+    it('returns an object containing all facets with their filters', () => {
```

Comment:
Change by `returns an object containing all the facets with their filters`

Reply:
done